webpack.config.js

```diff
@@ -52,22 +52,22 @@ switch (PUBLISH) {
     // build for extension injection
     PATH_VERSION = ''
     SUBVERSION = 'EXTENSION'
-    PUBLIC_PATH = 'http://localhost:3333/build/'
-    MAP_PATH = '\n//# sourceMappingURL=http://bam-test-1.nr-local.net:3333/build/[url]'
+    PUBLIC_PATH = 'http://localhost:3333/dist/cdn/'
+    MAP_PATH = '\n//# sourceMappingURL=http://bam-test-1.nr-local.net:3333/dist/cdn/[url]'
     break
   case 'NPM':
     // build for extension injection
     PATH_VERSION = ''
     SUBVERSION = 'NPM'
     PUBLIC_PATH = '/dist/'
-    MAP_PATH = '\n//# sourceMappingURL=http://bam-test-1.nr-local.net:3333/build/[url]'
+    MAP_PATH = '\n//# sourceMappingURL=http://bam-test-1.nr-local.net:3333/dist/cdn/[url]'
     break
   default:
     // local build
     PATH_VERSION = ''
     SUBVERSION = 'LOCAL'
-    PUBLIC_PATH = '/build/'
-    MAP_PATH = '\n//# sourceMappingURL=http://bam-test-1.nr-local.net:3333/build/[url]'
+    PUBLIC_PATH = '/dist/cdn/'
+    MAP_PATH = '\n//# sourceMappingURL=http://bam-test-1.nr-local.net:3333/dist/cdn/[url]'
```

Comment:
Maybe related to the npm size thread, can local be kept in a local `/build` dir?